packages/fluentui/docs/src/prototypes/chatPane/services/messageFactoryMock.tsx

```diff
@@ -13,6 +14,7 @@ import * as React from 'react';
 import * as _ from 'lodash';
 import * as keyboardKey from 'keyboard-key';
 import { ChatData, UserStatus, MessageData, UserData, areSameDay, getFriendlyDateString } from '.';
+import { DownloadIcon } from '@fluentui/react-icons-northstar';
```

Comment:
```suggestion
import { AcceptIcon, DownloadIcon } from '@fluentui/react-icons-northstar';
```
For AcceptIcon suggestion below